oned/bench/chunks_bench.cc

```diff
@@ -0,0 +1,44 @@
+#include <benchmark/benchmark.h>
+#include "oned/chunks.hpp"
+#include <vector>
+
+// Function to generate a test sequence
+std::vector<int> GenerateTestSequence(size_t size) {
+  std::vector<int> data(size);
+  for (size_t i = 0; i < size; ++i) {
+    data[i] = static_cast<int>(i); // Simple increasing sequence
+  }
```

Comment:
Optional: You can replace this for loop with:
```suggestion
  std::iota(data.begin(), data.end(), 0);  // Simple increasing sequence
```

You'll need a `#include <numeric>` at the top.

Reply:
Thanks a lot, First time learning this way